Must find action functions from script

Hello Voice Chat Users!
If you're recieveing an error like this:

```log
[01:46:36 ERROR]: [com.sedmelluq.discord.lavaplayer.track.playback.LocalAudioTrackExecutor] Error in playback of LyFpwF_m8-g
com.sedmelluq.discord.lavaplayer.tools.FriendlyException: Something broke when playing the track.
	at SimpleVoiceChatMusic-2.0.1.jar/com.sedmelluq.discord.lavaplayer.tools.ExceptionTools.wrapUnfriendlyExceptions(ExceptionTools.java:45) ~[SimpleVoiceChatMusic-2.0.1.jar:?]
	at SimpleVoiceChatMusic-2.0.1.jar/com.sedmelluq.discord.lavaplayer.track.playback.LocalAudioTrackExecutor.execute(LocalAudioTrackExecutor.java:121) ~[SimpleVoiceChatMusic-2.0.1.jar:?]
	at SimpleVoiceChatMusic-2.0.1.jar/com.sedmelluq.discord.lavaplayer.player.DefaultAudioPlayerManager.lambda$executeTrack$2(DefaultAudioPlayerManager.java:339) ~[SimpleVoiceChatMusic-2.0.1.jar:?]
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144) ~[?:?]
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642) ~[?:?]
	at java.base/java.lang.Thread.run(Thread.java:1583) ~[?:?]
Caused by: java.lang.IllegalStateException: Must find action functions from script: /s/player/6450230e/player_ias.vflset/en_US/base.js
	at SimpleVoiceChatMusic-2.0.1.jar/dev.lavalink.youtube.cipher.SignatureCipherManager.extractFromScript(SignatureCipherManager.java:279) ~[SimpleVoiceChatMusic-2.0.1.jar:?]
	at SimpleVoiceChatMusic-2.0.1.jar/dev.lavalink.youtube.cipher.SignatureCipherManager.getCipherScript(SignatureCipherManager.java:238) ~[SimpleVoiceChatMusic-2.0.1.jar:?]
	at SimpleVoiceChatMusic-2.0.1.jar/dev.lavalink.youtube.clients.skeleton.NonMusicClient.loadTrackInfoFromInnertube(NonMusicClient.java:110) ~[SimpleVoiceChatMusic-2.0.1.jar:?]
	at SimpleVoiceChatMusic-2.0.1.jar/dev.lavalink.youtube.clients.skeleton.StreamingNonMusicClient.loadFormats(StreamingNonMusicClient.java:35) ~[SimpleVoiceChatMusic-2.0.1.jar:?]
	at SimpleVoiceChatMusic-2.0.1.jar/dev.lavalink.youtube.track.YoutubeAudioTrack.loadBestFormatWithUrl(YoutubeAudioTrack.java:208) ~[SimpleVoiceChatMusic-2.0.1.jar:?]
	at SimpleVoiceChatMusic-2.0.1.jar/dev.lavalink.youtube.track.YoutubeAudioTrack.processWithClient(YoutubeAudioTrack.java:145) ~[SimpleVoiceChatMusic-2.0.1.jar:?]
	at SimpleVoiceChatMusic-2.0.1.jar/dev.lavalink.youtube.track.YoutubeAudioTrack.process(YoutubeAudioTrack.java:93) ~[SimpleVoiceChatMusic-2.0.1.jar:?]
	at SimpleVoiceChatMusic-2.0.1.jar/com.sedmelluq.discord.lavaplayer.track.playback.LocalAudioTrackExecutor.execute(LocalAudioTrackExecutor.java:109) ~[SimpleVoiceChatMusic-2.0.1.jar:?]
	... 4 more
	Suppressed: dev.lavalink.youtube.ClientInformation: 
  yts.version: 1.12.0
  client.identifier: ANDROID_VR
  client.options: ClientOptions{playback=true, playlistLoading=true, videoLoading=true, searching=true}
```

A new error has come about we have noticed. **This is happening with everyone and we understand.** This is an issue with lavalink player in terms of youtube (most likely rewriting their API) we will have an update as soon as they release a new version. Thank you!

You can track more in-depth if you'd like here: https://github.com/lavalink-devs/youtube-source/issues/129

--- a/pom.xml
+++ b/pom.xml
@@ -6,7 +6,7 @@
 
     <groupId>dev.klash</groupId>
     <artifactId>SimpleVoiceChatMusic</artifactId>
-    <version>2.0.1</version>
+    <version>3.0.0</version>
     <packaging>jar</packaging>
 
     <name>SimpleVoiceChatMusic</name>
@@ -94,7 +94,7 @@
         <dependency>
             <groupId>dev.lavalink.youtube</groupId>
             <artifactId>v2</artifactId>
-            <version>1.12.0</version>
+            <version>1.13.0</version>
         </dependency>
         <dependency>
             <groupId>org.jsoup</groupId>

